modules/docker/Makefile

```diff
@@ -18,6 +18,13 @@ TARGET_VERSION_LATEST ?= $(SEMVERSION_MAJOR_MINOR)-latest
 
 DOCKER_IMAGE_LATEST ?= $(TARGET_DOCKER_REGISTRY)/$(IMAGE_NAME):$(TARGET_VERSION_LATEST)
 
+OPERATOR_YAML := deploy/operator.yaml
+TAG_FOR_DEV := $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
We already have various git related variables available to us here: https://github.com/multicloud-ops/build-harness-extensions/blob/master/modules/semver/Makefile
including `SEMVERSION_GIT_CANONICAL` which expands to `$(shell $(GIT) describe --always  --abbrev=10 --dirty)` and will include a dirty-hash in non CI environments via
```
SEMVERSION_CANONICAL_DEVHASH=$(shell echo $(shell git add -u && git write-tree && git reset) | cut -c-10)
```

Could we use that?

Reply:
Yes I could definitely use that - didn't know exactly what to use/where to look to find what was available